### PR TITLE
[SW-713] Pop up elements overlap when swapping tokens on dex and the label is missing

### PR DESCRIPTION
**Pull request by pstachel**

### CODE REVIEW

File: src/popup/components/Modals/ConfirmTransactionSign.vue [hunk 1]
```diff
@@ -194,6 +197,12 @@ export default defineComponent({
     const getTxSymbol = useGetter('getTxSymbol');
     const getTxAmountTotal = useGetter('getTxAmountTotal');
 
+    const tokensLoaded = computed(
+      () => (isDex.value || isAllowance.value)
```
Comment (peronczyk) [inline src/popup/components/Modals/ConfirmTransactionSign.vue:201]: Right now this returns `number | false` but should return boolean I guess.

**Review by peronczyk — APPROVED** [review]

File: src/popup/components/Modals/ConfirmTransactionSign.vue (hunk 1 above)
Comment (pstachel) [reply src/popup/components/Modals/ConfirmTransactionSign.vue:201]: You're right, I'll change it

Comment (subhod-i) [reply src/popup/components/Modals/ConfirmTransactionSign.vue:201]: Could change it to `isTokensLoaded`

File: src/popup/components/Modals/ConfirmTransactionSign.vue [hunk 2]
```diff
@@ -4,82 +4,85 @@
...
-          :amount="tokenAmount"
-          :symbol="tokenSymbol"
-          :aex9="isTransactionAex9(txWrapped)"
-          :hide-fiat="!swapTokenAmountData.isAe"
-          data-cy="total"
-        />
+        <template #value>
+          <TokenAmount
+            :amount="nameAeFee"
+          />
```
Comment (konradodo) [inline src/popup/components/Modals/ConfirmTransactionSign.vue:35]:
```suggestion
          <TokenAmoun :amount="nameAeFee" />
```

File: src/popup/components/Modals/ConfirmTransactionSign.vue [hunk 3]
```diff
@@ -253,16 +256,24 @@ export default defineComponent({
...
         { tx: { ...txParams, ...props.transaction } } as ITransaction,
         availableTokens.value,
       )?.tokens;
-      if (!isPool.value) return tokens;
-      if (isProvideLiquidity.value) return tokens.filter((t) => !t.isPool);
+      if (!isPool.value) {
+        return tokens;
+      }
+      if (isProvideLiquidity.value) {
+        return tokens.filter((t) => !t.isPool);
```
Comment (konradodo) [inline src/popup/components/Modals/ConfirmTransactionSign.vue:275]:
```suggestion
        return tokens.filter((token) => token.isPool);
```

**Review by konradodo — APPROVED** [review]

**Pull request merged**